.eslintrc

```diff
@@ -32,6 +32,7 @@
     "camelcase": [2, {
       "properties": "never"
     }],
+   "no-console": [0] //allow use of javascript console in tests.
```

Comment:
also if you'd like to violate a rule in one place eslint lets you temporarily disable a rule with a comment (similar to pylint). see the docs: http://eslint.org/docs/user-guide/configuring#configuring-rules